websites/M/MusicCast/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "Marino",
+    "id": "598953747963707409"
+  },
+  "url": "musiccast.marinosite.xyz",
+  "description": {
+    "en": "Listen to the best tunes 24/7"
+  },
+  "service": "MusicCast",
+  "version": "1.0.0",
+  "logo": "https://cdn.marinosite.xyz/ps0t6c46.png",
+  "thumbnail": "https://cdn.marinosite.xyz/rxtizr28.png",

```

Comment:
You will have to up the resolution of the thumbnail or make it less blurry.

Reply:
it is intentional

Reply:
It is intentional, however, I can remove the blur of you wish. I will be doing so.